flink-formats/flink-avro/src/main/java/org/apache/flink/formats/avro/AvroRowSerializationSchema.java

```diff
@@ -157,7 +157,7 @@ public int hashCode() {
 
     // --------------------------------------------------------------------------------------------
 
-    private GenericRecord convertRowToAvroRecord(Schema schema, Row row) {
+    public GenericRecord convertRowToAvroRecord(Schema schema, Row row) {
```

Comment:
I think we should factor that to some utility class instead. It feels unnatural that the `ParquetInputFormat` needs to have an instance of `AvroRowSerializationSchema` and we need to expose this previously private method across packages. However, if we need state for this method, I can be persuaded to keep as is.

Reply:
great suggestion, I was a bit too quick with the instance ! Thanks ! Refactoring to static bunch of methods (basically convert* methods) works so no state needed. I'll extract them to a util class in flink-avro package (as these methods depend on avro) and refer it in flink-avro/AvroRowSerializationSchema and flink-parquet/ParquetAvroInputFormat. But the module dep will remain.

Reply:
I extracted conversion methods to a new util called _AvroConversions_ in flink-avro module

I isolated this change into a different commit. Indeed, IMHO we should keep this commit separated for isolation/revert purposes as it concerns a different module (flink-avro). I found that it was not necessary to open a different ticket just for a refactoring such as this. A separate commit seems enough.